doc/sphinx/users-guide/vcl-grace.rst

```diff
@@ -121,8 +117,9 @@ you can check if the backend is sick and modify the behavior when it
 comes to grace. There are essentially two ways of doing this. You can
 explicitly deliver kept object (that is not within grace) when you see
 that the backend is sick, or you can explicitly `not` serve an expired
-object when you know that the backend is healthy. The two methods have
-slightly different characteristics, as we shall see.
+object when you know that the backend is healthy. At the current time,
+we only recommend the first strategy, as the behavior of `keep` will
+change with the next major release of Varnish Cache.
```

Comment:
Have I heard about how keep changes ?

Reply:
Good point. Of course, the change is that *keep* objects cannot be delivered any more. If the last commit in the PR is pushed, then only the first method will work. I can move the documentation patch last if I get the other ones OK'ed, and update the text such that only the first strategy is left.